proxy.go

```diff
@@ -74,6 +99,9 @@ func ProxyHandler(w http.ResponseWriter, req *http.Request) {
 			w.Write([]byte(err.Error()))
 			return
 		}
+		// Close the connection to reuse it
+		defer r.Body.Close()
```

Comment:
Oh nevermind. This is a different response. Is it common Go practice to re-use single letter variables for different purposes? I find it particularly confusing.